src/ros_filter_utilities.cpp

```diff
@@ -98,14 +98,15 @@ namespace RosFilterUtilities
                            const std::string &targetFrame,
                            const std::string &sourceFrame,
                            const ros::Time &time,
+                           const ros::Duration &timeout,
```

Comment:
Just reading back over this again. I'm a bit concerned that this method will cause an ABI breakage and will also mean that anyone who may be using this method will have to change their source. While I sincerely doubt there are any users who are actually using any of these methods, I think I'd prefer that we add a new overload of this method that has this signature, and keep the old signature intact. You can even make it so that this method simply calls the new one with a 0 timeout parameter.